llvm/include/llvm/IR/VectorBuilder.h

```diff
@@ -100,11 +99,11 @@ class VectorBuilder {
                                  const Twine &Name = Twine());
 
   /// Emit a VP reduction intrinsic call for recurrence kind.
-  /// \param Kind        The kind of recurrence
+  /// \param RdxID       The intrinsic id of llvm.vector.reduce.*
```

Comment:
The majority of comments in `llvm/` use `intrinsic ID` as opposed to `intrinsic id`, so I think we should use that

Reply:
a96aa388ba302a42b7de6db3c78f6492b3bafc2c